lib/swig/GeNNModel.py

```diff
@@ -45,16 +47,11 @@ def __init__(self, precision=None, modelName='GeNNModel', enableDebug=False,
         
         self._built = False
         self._cpuOnly = cpuOnly
-        if not cpuOnly:
-            self._localhost = pg.init_cuda_mpi()
-        else:
-            self._localhost = 0
-
+        self._localhost = pg.initMPI_pygenn()
         
         pg.setDefaultVarMode( pg.VarMode_LOC_HOST_DEVICE_INIT_DEVICE )
-        pg.cvar.debugCode = enableDebug
-        pg.cvar.autoInitSparseVars = autoInitSparseVars
-        pg.cvar.buildSharedLibrary = True
+        pg.GeNNPreferences.cvar.debugCode = enableDebug
+        pg.GeNNPreferences.cvar.autoInitSparseVars = autoInitSparseVars
```

Comment:
This behaviour is also purely legacy to retain compatibility with old models- just set it to True